### DISCUSSION TITLE
Is there any advantage using next/image for .svg?

### DISCUSSION THREAD

**Discussion by Nases** [vercel/next.js#35641] [Help]
Is there any advantage using next/image over native img for .svg?

**Comment by 0x15BA88FF** [comment]
Hi @Nases,
well yeah there is, using next/image automatically resizes the graphic then the screen size reduces making it more responsive and I think it also comes with the lazy loading

**Reply by transitive-bullshit** [reply]
No; svg images are vector graphics which are size independent. Unlike raster images like png/jpg/bmp/etc, they don't have an intrinsic size, and therefore 99% of next/image functionality doesn't apply to them.

The lazy loading is the only real advantage I can see, but you can get this with `<img loading='lazy' />` without using `next/image`.

**Reply by 0x15BA88FF** [reply]
oh sorry for the miss-information 😅, but overall I do recommend using next/image when creating next apps, they have other props for images like [placeholders](https://nextjs.org/docs/api-reference/next/image#placeholder) which can be given a blur while loading, style, [objectFit](https://nextjs.org/docs/api-reference/next/image#objectfit), [objectPosition](https://nextjs.org/docs/api-reference/next/image#objectposition), [lazyBoundary](https://nextjs.org/docs/api-reference/next/image#lazyboundary).. states like [onLoadingComplete](https://nextjs.org/docs/api-reference/next/image#onloadingcomplete) and more handy stuff. it is also good to keep your code consistent but it is your choice to pick

**Reply by todd-kunda** [reply]
what about calling svg as component then adding a className so that we can change it's color or sizes?

**Comment by lachlanjc** [comment]
I think the main thing is consistency with other types of images you're using next/image for, & if you were going to be using an `<img>` tag otherwise, the container sizing to avoid CLS. Not worth the setup in my personal opinion, but the compatibility can be handy, especially if you don't know what file type is coming from a CMS etc.
